addon/components/g-map/marker.js

```diff
@@ -15,6 +15,6 @@ export default class Marker extends TypicalMapComponent {
   }
 
   newMapComponent(options = {}) {
-    return new google.maps.Marker(options);
+    return new google.maps.marker.AdvancedMarkerElement(options);
```

Comment:
How backwards-compatible is this? If someone is using an older version of google maps, will this break? Maybe it should be a separate component then?

Reply:
Yes. I think we might as well make it a separate component.